projects/swimlane/ngx-charts/src/lib/models/chart-data.model.ts

```diff
@@ -36,6 +36,24 @@ export interface PieGridDataItem extends DataItem {
   value: number;
 }
 
+export interface PieGridData {
+  data: PieGridDataItem;
+  height: number;
+  width: number;
+  x: number;
+  y: number;
+}
+
+export interface PieArc {
```

Comment:
@blogcraft Was this `PieArc` interface also giving you problems? If not, I'd leave it where it was (at least for now) as it appears it is not imported anywhere else.

Reply:
Pie Arc needs to be moved too, if not the Cycle error persists:

pie-grid.component.ts -> pie-grid-series.component.ts -> pie-grid.component.ts